.github/workflows/testing-and-deployment.yml

```diff
@@ -136,14 +136,22 @@ jobs:
         include:
           - python-version: "3.8"
             vtk-version: "9.0.3" # Requires numpy<1.24
+            numpy-version: "default"
           - python-version: "3.9"
             vtk-version: "9.1"
+            numpy-version: "default"
           - python-version: "3.10"
             vtk-version: "9.2.2"
+            numpy-version: "default"
           - python-version: "3.11"
             vtk-version: "9.2.6"
+            numpy-version: "default"
           - python-version: "3.12"
             vtk-version: "latest"
+            numpy-version: "default"
+          - python-version: "3.12"
+            vtk-version: "latest"
+            numpy-version: "latest"
```

Comment:
Hrm... I had to iterate on this one more time but I'm a little concerned about the difference in meaning of "latest" for `vtk-version` and `numpy-version`. "latest" for VTK means the latest stable release where as "latest" for numpy here means pre-release. Should we change the terms here so its consistent for numpy?

Reply:
Sorry for my onslaught of nitpicky feedback here! 😄

Reply:
I'm open to feedback, but we will have to change this once numpy fully releases in any cases here.  So whatever we choose will be short lived.  At that point and assuming we can fix the errors, we should generally test with latest numpy going forward, and targeted older versions.

But I take the point that the way GitHub displays the making makes it hard to distinguish which test is what.  Let's use 'pre' for numpy, or something like this, for now.